components/PageData.js

```diff
@@ -18,17 +18,17 @@ const PageContent = ({ children }) => (
 
 const PageData = ({
   isLight, isDone, disableWhenDone, backgroundColor, image,
-  title, subtitle, notice, action, next, end, currentPage, pages, ...rest
+  title, subtitle, notice, action, next, end, currentPage, pages, font, ...rest
 }) => (
   <Page {...rest}>
     <PageContent>
       <View style={styles.image}>
         {image}
       </View>
-      <Text style={{ ...styles.title, ...(isLight ? styles.titleLight : {}) }}>
+      <Text style={{ ...styles.title, ...(isLight ? styles.titleLight : {}), ...(font ? font : {}) }}>
```

Comment:
When unpacking the arguments use a default parameter `{.. currentPage, pages, font={}, ... rest` then this won't be needed (just put `font` here)